records_mover/db/bigquery/load_job_config_options.py

```diff
@@ -324,12 +324,14 @@ def add_load_job_csv_config(unhandled_hints: Set[str],
     quiet_remove(unhandled_hints, 'datetimeformat')
 
     if hints.datetimeformattz in ['YYYY-MM-DD HH:MI:SSOF',
+                                  'YYYY-MM-DD HH24:MI:SSOF',
                                   'YYYY-MM-DD HH:MI:SS']:
         pass
     else:
         cant_handle_hint(fail_if_cant_handle_hint, 'datetimeformattz', hints)
     quiet_remove(unhandled_hints, 'datetimeformattz')
 
+    # TODO: Ensure this raises as HH:MI:SS should also work
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker